src/derivatives/introduction.md

```diff
@@ -4,9 +4,16 @@ Derivatives are outputs of common processing pipelines, capturing data and
 meta-data sufficient for a researcher to understand and (critically) reuse those
 outputs in subsequent processing.
 Standardizing derivatives is motivated by use cases where formalized
-machine-readable access to processed data enables higher level processing.
+machine-readable access to processed data enables higher-level processing.
+
+The following sections cover additions to and divergences from "raw" BIDS. Raw data are
+data that have been curated to BIDS, i.e. a BIDS “raw” dataset does not have another BIDS
+dataset as a source. If you are in doubt if your dataset is "raw" or
+is "derivatives" ask yourself the following question: "Are my data derived from at least
+one BIDS valid dataset?" if the answer is yes, then this is a BIDS derivative dataset.
```

Comment:
I think there might be need to refactor a little to not step into murky waters of derived "data" vs "dataset". 
e.g. here if to go to minimal changes for that purposes I would at least suggest
```suggestion
is "derivatives" ask yourself the following question: "Are my data derived from at least
one other valid BIDS dataset?" if the answer is yes, then this is a BIDS derivative dataset.
```
to accent on the fact that e.g. defaced images could be coming from the same BIDS dataset but we would not call it derived. Moreover some minimally preprocessed/converted data might be based on data in the same bids dataset.

Reply:
I even wonder if we could/should provide some examples, like
- if files are minimally processed "within" the BIDS dataset (e.g. "defaced"), the dataset remains "raw"
- if a new dataset produced based on another BIDS dataset, and even if it has minimal or no changes to the files but comes from another BIDS dataset - it is a "derived dataset".